src/services/ZapperService.ts

```diff
@@ -33,4 +37,39 @@ export class ZapperService {
       return 0;
     }
   }
+
+  //TODO(adotcruz): Implement function that gets all the tokens in a user wallet.
+
+  //This function gets all the tokens a user has lended in aave, as well as each respective APY.
+  //
+  // By default Protocol is set to Aave, and Network is set to Polygon, but this can be made
+  // more generic as more support is added to different protocols/networks.
+  static async GetUserAaveHoldings(
+    address: string,
+    protocol: ZapperSupportedProtocolWalletBalances = ZapperSupportedProtocolWalletBalances.Aave,
+    network: ZapperNetworks = ZapperNetworks.polygon
+  ): Promise<ZapperAaveBalance[]> {
+    // Address must always be treated as lowercase for some functions.
+    const standardizedAddress = address.toLowerCase();
+    const endpoint = `v1/protocols/${protocol}/balances`;
+    const params = `?addresses%5B%5D=${standardizedAddress}&network=${network}&api_key=${ZapperApiKey}`;
+    try {
+      const userAaveResponse = (await HttpService.get<ZapperAaveBalanceResponse>(
+        `${ZapperEndpoint}/${endpoint}${params}`
```

Comment:
you may already have looked into this, but is there a param we can pass into the request to filter by aave version instead of having to filter after fetching the response?

Reply:
lol their response by default returns a `[]` when there should technically only be 1 entry. This was just in case the API ever changes, and T thought it make's it slightly easier to read what is being fetched from the array.

ex: https://api.zapper.fi/v1/protocols/aave-v2/balances?addresses%5B%5D=0x4B853a4Cedae52b8fC8370951Ab19563C1399012&network=polygon&api_key=96e0cc51-a62e-42ca-acee-910ea7d2a241

Reply:
sorry I meant the `aaveProducts.label == ZapperProtocolBalanceProductLabel.AaveV2` filtering part

Reply:
no :( `aave-v2` is already in the url (i pasted the example above) -- it's just how their API is designed that's why I said `the response by default returns a [] when there should technically only be 1 entry` -- doesn't make sense tho :(